Assets/Scripts/Game/Enemy/EnemyWaves.cs

```diff
@@ -4,8 +4,8 @@
 [System.Serializable]
 public class EnemyWave
 {
-    public int horizonalEnemyNumber;
-    public int verticalEnemyNumber;
+    public int horizonalEnemyNumber = 1;
+    public int verticalEnemyNumber = 1;
     public Vector3 spawnPoint;
     public Vector3 interval;
     public GameObject enemyType;
```

Comment:
この辺りをプロパティにした方がいいらしい(メンバ変数直接いじらない方がいいみたいで)
```suggestion
    public Vector3 spawnPoint{get; set;}
    public Vector3 interval{get; set;}
    public GameObject enemyType{get; set;}
```